frontend/src/components/header/logo/logo.tsx

```diff
@@ -1,13 +1,17 @@
 import React from 'react';
+import { Link } from 'react-router-dom';
 
 import LogoIcon from '@assets/images/logo.svg';
+import { AppRoute } from '@common/enums/enums';
 
 import styles from './logo.module.scss';
 
 export const Logo: React.FC = () => {
   return (
     <div className={styles.logoContainer}>
-      <img src={LogoIcon} alt="logo" />
+      <Link to={AppRoute.ROOT}>
+        <img src={LogoIcon} alt="logo" />
+      </Link>
     </div>
```

Comment:
This change works, but it's not part of the font fix. Please keep PRs as atomized as possible